cmd/server/main.go

```diff
@@ -256,6 +267,7 @@ func newProxyRunOptions() *ProxyRunOptions {
 		serverPort:             8090,
 		agentPort:              8091,
 		adminPort:              8092,
+		healthPort:             8093,
```

Comment:
Sorry just a small nit, can we keep the healthPort at 8092 and move the adminPort to 8093 as the default? This reduces the amount of existing readiness/liveness probe code that needs to be changed.

Reply:
:heavy_check_mark: